docs/core/compatibility/globalization/6.0/culture-creation-invariant-mode.md

```diff
@@ -5,7 +5,7 @@ ms.date: 07/23/2021
 ---
 # Culture creation and case mapping in globalization-invariant mode
 
-This breaking change affects *globalization-invariant mode* in two ways:
+This breaking change affects *[globalization-invariant mode](https://tools.ietf.org/rfc/bcp/bcp47.txt)* in two ways:
```

Comment:
I don't think this is the correct link.

Reply:
The same link is used below, what's incorrect about it?

Reply:
It's a link to an RFC called "Tags for Identifying Languages". I think something like https://github.com/dotnet/runtime/blob/main/docs/design/features/globalization-invariant-mode.md would make a lot more sense.